File: A2/java/src/StrangePlanet.java
import java.util.*;

public class StrangePlanet {
	public static void main(String[] args) {
		String dataFile = "input/map-basic.txt";
		// String dataFile = "input/map-extension.txt";
		Engine eng = new Engine(dataFile);
		eng.run();
	}
}

File: A2/python-skeleton/StrangePlanet.py
import sys 

if __name__ == '__main__':
	if len(sys.argv) > 1:
		if sys.argv[1] == "basic":
			from Engine import Engine
			eng = Engine('map-basic.txt')
			if eng != None:
				eng.run()
		elif sys.argv[1] == "extension":
			from NewEngine import NewEngine
			eng = NewEngine('map-extension.txt')
			if eng != None:
				eng.run()
	else:
		print("usage: python3 StrangePlaent.py [basic/extension]")